test/openAPI/features/wallet_binding.feature

```diff
@@ -0,0 +1,31 @@
+@method=POST @endpoint=/wallet-binding
+Feature: The endpoint to validate wallet create wallet user id
+
+  @smoke @unit @positive
+  Scenario: Successfully validates the wallet and generates the wallet user id smoke type test
+    Given Wants to validate the wallet and generate wallet user id
+    When Send POST /wallet-binding request with given "n3fy2qkg9r7h2" as individualId and "OTP" as authFactorType and "encoded-json" as format and "string" as challenge and publicKey
```

Comment:
An implementation can support only a subset of authFactorTypes from the enum specificied, can we take the value from a configuration for  authFactorType inside challengeList?

Reply:
External authFactorType should be WLA

Reply:
I see the verification is added on response, but this comment is about how do we make the request. Can we take this as input like we take the transaction id ? we can call it supportedAuthFactorType

Reply:
If there is a associated task and if it is tracked, we can close this

Reply:
@vishwa-vyom I created the ticket: https://govstack-global.atlassian.net/browse/TECH-782
Feel free to edit the description if it's needed